afs-ws/afs-ws-server-utils/src/main/java/com/powsybl/afs/ws/server/utils/JwtTokenNeededFilter.java

```diff
@@ -1,76 +0,0 @@
-/**
- * Copyright (c) 2018, RTE (http://www.rte-france.com)
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/.
- */
-package com.powsybl.afs.ws.server.utils;
-
-import com.powsybl.commons.config.PlatformConfig;
-import io.jsonwebtoken.Claims;
-import io.jsonwebtoken.Jws;
-import io.jsonwebtoken.Jwts;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import javax.annotation.Priority;
-import javax.inject.Inject;
-import javax.ws.rs.NotAuthorizedException;
-import javax.ws.rs.Priorities;
-import javax.ws.rs.container.ContainerRequestContext;
-import javax.ws.rs.container.ContainerRequestFilter;
-import javax.ws.rs.core.HttpHeaders;
-import javax.ws.rs.core.Response;
-import javax.ws.rs.ext.Provider;
-import java.security.Key;
-
-/**
- * @author Ali Tahanout <ali.tahanout at rte-france.com>
- */
-@Provider
-@JwtTokenNeeded
-@Priority(Priorities.AUTHENTICATION)
-public class JwtTokenNeededFilter implements ContainerRequestFilter {
```

Comment:
I think we should not remove that filter, because the annotation `@JwtTokenNeeded` is used on existing java EE components that will remain in the repo.

Reply:
agreed

Reply:
I restored the file